example-completeness/src/LocalizableNormalizer.js

```diff
@@ -0,0 +1,24 @@
+import _ from 'lodash';
+import jp from 'jsonpath';
+
+exports.getLocalizations = function (product, path) {
+    var localizations = [];
+    const localizable = jp.query(product, path)[0];
+    if (typeof localizable === 'object') {
+        Object.keys(localizable).forEach(key => {
+            localizations.push(key);
+        });
+    }
+    return _.uniq(localizations)
+}
+
+exports.normalize = function (object) {
+    const localization = this.localization;
+    var obj = {};
+    if (typeof object != "string") {
```

Comment:
Do we need this conditional if we're checking for typeof object === 'object'  below?

Reply:
https://github.com/commercetools/commercetools-serverless-examples/pull/1/commits/de60fd91e57cb64fd31f7b70bcb6eddb2ec8aa9c